doc/classes/PhysicsServer2D.xml

```diff
@@ -434,6 +434,34 @@
 				Creates a physics body.
 			</description>
 		</method>
+		<method name="body_enable_shape">
```

Comment:
Same as `enable_shape_owner`, it would be more explicit with something like `body_set_shape_enabled`, without the default value.